package.json

```diff
@@ -43,7 +43,8 @@
     "publish:unstable": "./scripts/publish_unstable.sh",
     "publish:release": "./scripts/publish.sh",
     "version": "conventional-changelog -p angular -i CHANGELOG.md -s && git add CHANGELOG.md",
-    "version:release": "yarn version --minor --message \"chore(release): publish\""
+    "version:release": "yarn version --minor --message \"chore(release): publish\"",
+    "version:release:major": "yarn version --major --message \"chore(release): publish\""
```

Comment:
would be make sense just remove `--minor` ? 

```
"version:release": "yarn version --message \"chore(release): publish\"",
```

in the RELEASE.md specify we need to pick accordingly?

```
version:release --[major|minor|patch]
```

Reply:
try to keep it consistent with our https://github.com/mattrglobal/opencore/blob/master/package.json#L34

Reply:
Agree with @colinyip's suggestion here, can we please update it @kenhuang ?

Reply:
updated